lib/scope.js

```diff
@@ -741,15 +775,32 @@ module.exports = {
     };
 
     let error_handlers = {};
-    let error_found = false;
+    let was_found = false;
     for ( let error_key in possible_errors ) {
       if ( possible_errors[ error_key ][0] ) {
         if ( env_vars.DEBUG ) { await scope.page.waitFor( 60 * 1000 ); }
-        error_found = true;
+        was_found = true;
         error_handlers[ error_key ] = possible_errors[ error_key ][0];
       }
     }
 
-    return { error_found, error_handlers };
-  },
+    return { was_found, error_handlers };
+  },  // Ends scope.checkForError()
+
+  throwPageError: async function ( scope, { error_handlers = {}, remaining_vars = {}, id = '' }) {
+    /* Throw a cucumber error with the error that appeared on the page. */
+    let was_system_error = error_handlers.system_error !== undefined;
```

Comment:
Can `error_handlers.system_error` be `false`? If so, should the comparison be `error_handlers.system_error === true` instead? 

(Looks like we're not using the error_handlers variable right now, so this might be a moot point.

Reply:
Really good point. Turned into #172.